test/textfile/issue-3550.py

```diff
@@ -23,21 +23,27 @@
 #
 
 __revision__ = "__FILE__ __REVISION__ __DATE__ __DEVELOPER__"
+"""
+Test for GH Issue 3550
+
+ You can't pass a Windows path in a value to be interpolated, 
```

Comment:
doesn't this refer to a different PR/issue?

Reply:
Yes. I changed the .in file to not have 3550 in the filename so was necessary to update that test to use the new filename.